src/include/sof/audio/dts/DtsSofInterface.h

```diff
@@ -54,18 +59,20 @@ typedef struct DtsSofInterfaceBufferConfiguration
 	unsigned int                sampleRate;
 	unsigned int                numChannels;
 	unsigned int                periodInFrames;
-	unsigned int                totalBufferLengthInBytes;
 } DtsSofInterfaceBufferConfiguration;
 
 typedef void* (*DtsSofInterfaceAllocateMemory)(
-	void* pMemoryAllocationContext,
-	unsigned int length,
-	unsigned int alignment);
+		void *pMemoryAllocationContext,
+		unsigned int length,
+		unsigned int alignment);
```

Comment:
It looks like wrong alignment but since it is a cherry-pick patch so may be fixed in another/next patch. Thanks